bin/component-playground.js

```diff
@@ -7,7 +7,8 @@ var path = require('path'),
 var rootPath = path.join(__dirname, '..'),
     playgroundPath = path.join(rootPath, 'component-playground');
 
-var compiler = webpack(require(path.join(playgroundPath, 'config')).webpack);
+var userConfig = require(path.join(playgroundPath, 'config'));
```

Comment:
This contains the user config plus all the Cosmos defaults, so let's call it just `config`.